File: Publify.StripeAPI/Models/CardModel.cs
namespace Publify.StripeAPI.Models
{
    public class CardModel
    {
        public string FullName { get; set; } = null!;

        public string Last4Digits { get; set; } = null!;

        public long ExpMonth { get; set; }

        public long ExpYear { get; set; }

        public string CvcCheck { get; set; } = null!;

        public bool IsDefaultPaymentMethod { get; set; }

        public string Brand { get; set; } = null!;
    }
}

File: FrontEnd/src/app/models/CardModel.ts
export class CardModel {
  fullName: string;
  last4Digits: string;
  expMonth: number;
  expYear: number;
  cvcCheck: string;
  isDefaultPaymentMethod: boolean;
  /**
   * possible values
   * American Express, Diners Club, Discover, Eftpos Australia, JCB, MasterCard, UnionPay, Visa
   *
   */
  brand: string;
}
